contents/handbook/growth/customer-support.md

```diff
@@ -39,24 +39,36 @@ Providing support is a responsibility shared across our team - we expect everyon
 
 This does mean sometimes that, especially when we are particularly busy, customer success can take a bit of a back seat.
 
-This is why, in addition, one person takes on the **[Support Hero](/handbook/engineering/support-hero)** role each two week sprint. This is a rotating responsibility, where the person involved spends a significant chunk of their time responding to support queries across Slack, email and Papercups, and sharing that feedback with the team and/or building features and fixes in response. We have found that each stint as Support Hero has thrown up a lot of really valuable feedback. 
+This is why, in addition, one person takes on the **[Support Hero](/handbook/engineering/support-hero)** role each two week sprint. This is a rotating responsibility, where the person involved spends a significant chunk of their time responding to support queries across Slack, email and Zendesk, and sharing that feedback with the team and/or building features and fixes in response. We have found that each stint as Support Hero has thrown up a lot of really valuable feedback. 
 
 ### Simple, lightweight tools
 
 We go to where our users are. That means **we respond in the same channel that they reached out to**, rather than trying to funnel them somewhere else.
 
-We use [Papercups](https://papercups.io/) as our internal platform to get an overview of our support requests. This ensures that we don't miss anyone, especially when their request is passed from one person to another at PostHog, or if they message us over the weekend. If customer success is part of your role, you should have received an invite to join as part of your onboarding - if you didn't, ask Charles. 
+We use [Zendesk Support](https://zendesk.com/) as our internal ticket management platform to get an overview of our support requests. We also use [Help](https://www.atlassian.com/software/halp) to easily create and manage Zendesk tickets straight from the PostHog Users Slack. This ensures that we don't miss anyone, especially when their request is passed from one person to another at PostHog, or if they message us over the weekend. If customer success is part of your role, you should have received an invite to join as part of your onboarding - if you didn't, ask Charles. 
```

Comment:
Should refer to Grace, not Charles now right? (Or should list out here who the admins are for each)